File: Proyectos C++ (4)/iva.cpp
#include <iostream>

using namespace std;
int des, vp, cp, st, iva;
int main ()
{

	cout<<"Programa que calcula el total a pagar en una caja registradora"<<endl;
	cout<<"Ingrese la cantidad de productos: "<<endl;
	cin>>cp;
	cout<<"Ingrese el valor de cada producto: "<<endl;
	cin>>vp;
	cout<<"¿Desea ingresar otro articulo? Si=1, No=2 "<<endl;
	cin>>des;
	st = vp * cp;
	while (des==1)
	{
		cout<<"Ingrese la cantidad de productos: "<<endl;
		cin>>cp;
		cout<<"Ingrese el valor de cada producto: "<<endl;
		cin>>vp;
		st = st + (cp*vp);
		cout<<"Desea ingresar otro producto? SI=1, No=2 "<<endl;
		cin>>des;
	}
	iva = st * 0.16;
	cout<<"Ingrese su forma de pago: tarjeta=3, efectivo=4 "<<endl;
	cin>>des;
	if(des==3)
	{
		cout<<"El valor total a pagar es: "<<st+iva<<"\n Y el valor total del iva es: "<<iva<<endl;
	}
	else
	{
		cout<<"El valor tota a pagar es: "<<st-iva<<"\n y el valor del iva es: "<<iva<<endl;
	}
	

}

File: Proyectos python (4)/iva.py
vp=0
cp=0
iva=0
des=0
st=0
print("Programa que va a dar a conocer el total a pagar en una caja registradora")
cp=(int(input("Ingrese la cantidad de articulos:\n")))
vp=(int(input("Ingrese el valor de cada producto:\n")))
des=(int(input("¿Desea ingresar otro articulo? Si=1, No=2\n")))
st=vp*cp
while des == 1:
    cp=(int(input("Ingrese la cantidad de articulos:\n")))
    vp=(int(input("Ingrese el valor de cada producto:\n")))
    st=st+cp*vp 
    des=(int(input("¿Desea ingresar otro articulo? Si=1, No=2\n")))

iva = st * 0.16
des=(int(input("Ingrese su forma de pago: Tarjeta=3, Efectivo=4\n")))
if (des==3):
    print ("El valor total a pagar es: ",st+iva)
else:
    print("El valor total a pagar es: ",st-iva)
